frontend/package.json

```diff
@@ -48,6 +48,7 @@
     "@patternfly/react-icons": "4.93.6",
     "@patternfly/react-styles": "4.92.6",
     "@patternfly/react-table": "4.112.39",
+    "@patternfly/react-topology": "^4.91.27",
```

Comment:
I cannot suggest the change cause it's down bellow, but it would be nice to add:

```json
    "@patternfly/react-topology": {
      "react": "^18.2.0"
    }
```

In the override section to disable the npm warnings.